bin/setup

```diff
@@ -1,29 +1,40 @@
 #!/usr/bin/env ruby
-require 'pathname'
+require "pathname"
+require "fileutils"
+
+include FileUtils
+
+ENV["RAILS_ENV"]               ||= ENV["PASSENGER_APP_ENV"]
+ENV["RAILS_RELATIVE_URL_ROOT"] ||= ENV["PASSENGER_BASE_URI"]
```

Comment:
Setting an environment variable to `nil` in Ruby unsets the environment variable. So if both `RAILS_ENV` and `PASSENGER_APP_ENV` are unset, then `RAILS_ENV` stays unset.